examples/functions/sentiment-analysis/index.ts

```diff
@@ -0,0 +1,39 @@
+import {createClient} from '@sanity/client'
+import {documentEventHandler} from '@sanity/functions'
+
+export const handler = documentEventHandler(async ({context, event}) => {
+  const client = createClient({
+    ...context.clientOptions,
+    apiVersion: 'vX',
+    useCdn: false,
+  })
+  const {data} = event
+  const {local} = context // local is true when running locally
+
+  try {
+    const result = await client.agent.action.generate({
+      noWrite: local ? true : false, // if local is true, we don't want to write to the document, just return the result for logging
```

Comment:
```suggestion
      noWrite: !!local, // if local is true, we don't want to write to the document, just return the result for logging
```
Shouldn't this be enough? Alternatively `Boolean(local)`.